_config/site.yml

```diff
@@ -74,15 +74,15 @@ profiles:
     base_url: http://docker:<%= ENV['AWESTRUCT_HOST_PORT'] %>/
     port: <%= ENV['AWESTRUCT_HOST_PORT'] %>
     dcp_base_url: http://searchisko:8080/
-    dcp_base_protocol_relative_url: //<%= (ENV.key?('ACCESSIBLE_SLAVE_IP') && !ENV['ACCESSIBLE_SLAVE_IP'].empty?) ? ENV['ACCESSIBLE_SLAVE_IP'] : 'docker' %>:<%= ENV['SEARCHISKO_HOST_PORT'] %>/
+    dcp_base_protocol_relative_url: //<%= ENV['ACCESSIBLE_SLAVE_IP'].nil? ? 'docker' : ENV['ACCESSIBLE_SLAVE_IP'] %>:<%= ENV['SEARCHISKO_HOST_PORT'] %>/
```

Comment:
My fix here is better I think. IIRC the nil? didn't work correctly on my machine.